notebook/static/notebook/js/notebook.js

```diff
@@ -2982,13 +2982,17 @@ define([
                 .text(i18n.msg._("here"))
             )
         );
-
+        var fb = '';
+        if(from_notification === 'Notification_button') {
```

Comment:
We've got two stages in this where we compare something against a hardcoded string and then use another hardcoded string:

- `'Notification_button'` -> `'notification_trusted'`
- `'notification_trusted'` -> `'#notification_trusted'`

It's probably better if we pass in the final selector string in the first place, and avoid the comparisons. Just if there's any `focus_button` set at the end, use it as a selector to focus.